Files/nginx.sh

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/bash
+#
+# Checking if Nginx is already installed, if not, it'll install it. Also has an option to uninstall cleanly.
+
+pkg=nginx
+echo "Nginx"
+    select yn in "Check-and-Install" "Delete"; do
+    case $yn in
+        Check-and-Install ) 
+		  rpm --query ${pkg}
+		if [ $? -eq 0 ]; then
+			echo "${pkg} is already installed"
+		else
+			echo -e "Installing ${pkg}...\n" \
```

Comment:
What is the reason behind concatenating multiple commands?

Reply:
The result of this bulk commands will be a proper installation of Nginx, when the system finds Nginx is not already installed it will run this bulk.
Because I choose to use cases in my code all those commands needs to be together as case bulk of action.